docs/source/sci_api_tutorial/preparation_remove_solvent.rst

```diff
@@ -0,0 +1,125 @@
+==============================================
+ Preparation: Remove Solvent
+==============================================
+
+Briefs
+==============================================
+
+This science API, named ``enzy_htp.preparation.clean.remove_solvent``,
+removes solvent from the supplied ``enzy_htp.structure.Structure`` class instance 
+(hereafter referred to as ``Structure`` instance).
+
+All ``enzy_htp.structure.Solvent`` class instances (hereafter referred to as
+``Solvent`` instances) will be removed.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/preparation_remove_solvent_dfd.svg
+        :width: 100%
+        :alt: preparation_remove_solvent
+
+    .. **input**: A ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+    .. **output**: A ``Structure`` instance with solvent removed (in-place modification,
+    .. but still provides a return value).
+
+
+Arguments
+==============================================
+
+``stru``
+    The input ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+.. ``protect``
+.. (optional) Protect some solvent from removal and change its rtype to Ligand. Use selection grammer.
+
+Examples
+==============================================
+
+Before Execution: Load Structure
+----------------------------------------------
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    In order to make use of the API, we should have structure loaded.
+
+    .. code:: python    
+
+        import enzy_htp.structure as struct
+                                    
+        sp = struct.PDBParser()
+
+        pdb_filepath = "/path/to/your/structure.pdb"
+        stru = sp.get_structure(pdb_filepath)
+
+Execute API
+----------------------------------------------
+
+Use ``preparation.clean.remove_solvent`` to remove solvent from your structure.
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    The way to use ``remove_solvent`` is pretty simple.
+
+    .. code:: python
+
+        from enzy_htp.preparation.clean import remove_solvent
+        
+        stru = remove_solvent(stru=stru)
```

Comment:
This is redundant. It should always be imported from preparation

Reply:
Redundant content has been removed.